backend/inspirehep/records/cli.py

```diff
@@ -192,3 +193,32 @@ def close_expired_jobs(notify):
             send_job_deadline_reminder(dict(job_record))
 
     LOGGER.info("Closed expired jobs", notify=notify, num_records=len(expired_jobs))
+
+
+@click.group()
+def relationships():
+    """Command to manage records relationships."""
+
+
+@relationships.command(help="Populates relations in JournalLiterature table")
+@with_appcontext
+def populate_journal_literature_table():
+    query = {
+        "query": {
+            "nested": {
+                "path": "publication_info",
+                "query": {
+                    "bool": {
+                        "must": [
+                            {"exists": {"field": "publication_info.journal_record"}}
+                        ]
+                    }
+                },
+            }
+        }
+    }
+    records = LiteratureSearch().from_dict(query).params().scan()
+
+    for record in records:
+        record = LiteratureRecord.get_record(record.meta.id)
+        record.update_journal_relations()
```

Comment:
Let's chunk them and add a try except in case something fails not to interrupt the process